app_dart/analysis_options.yaml

```diff
@@ -42,155 +42,11 @@ analyzer:
-    - prefer_final_fields
-    # - prefer_final_in_for_each # not yet tested
-    - prefer_final_locals
-    # - prefer_for_elements_to_map_fromIterable # not yet tested
-    - prefer_foreach
-    # - prefer_function_declarations_over_variables # not yet tested
-    - prefer_generic_function_type_aliases
-    # - prefer_if_elements_to_conditional_expressions # not yet tested
-    - prefer_if_null_operators
-    - prefer_initializing_formals
-    - prefer_inlined_adds
-    # - prefer_int_literals # not yet tested
-    # - prefer_interpolation_to_compose_strings # not yet tested
-    - prefer_is_empty
-    - prefer_is_not_empty
-    - prefer_iterable_whereType
-    # - prefer_mixin # https://github.com/dart-lang/language/issues/32
-    # - prefer_null_aware_operators # disable until NNBD, see https://github.com/flutter/flutter/pull/32711#issuecomment-492930932
-    - prefer_relative_imports
-    - prefer_single_quotes
-    - prefer_spread_collections
-    - prefer_typing_uninitialized_variables
-    - prefer_void_to_null
-    # - provide_deprecation_message # not yet tested
-    # - public_member_api_docs # enabled on a case-by-case basis; see e.g. packages/analysis_options.yaml
-    - recursive_getters
-    - slash_for_doc_comments
-    # - sort_child_properties_last # not yet tested
-    - sort_constructors_first
-    - sort_pub_dependencies
-    - sort_unnamed_constructors_first
-    - test_types_in_equals
-    - throw_in_finally
-    # - type_annotate_public_apis # subset of always_specify_types
-    - type_init_formals
-    - unawaited_futures
-    # - unnecessary_await_in_return # not yet tested
-    - unnecessary_brace_in_string_interps
-    - unnecessary_const
-    - unnecessary_getters_setters
-    # - unnecessary_lambdas # has false positives: https://github.com/dart-lang/linter/issues/498
-    - unnecessary_new
-    - unnecessary_null_aware_assignments
-    - unnecessary_null_in_if_null_operators
-    - unnecessary_overrides
-    - unnecessary_parenthesis
-    - unnecessary_statements
-    - unnecessary_this
-    - unrelated_type_equality_checks
-    # - unsafe_html # not yet tested
-    - use_full_hex_values_for_flutter_colors
-    # - use_function_type_syntax_for_parameters # not yet tested
-    - use_rethrow_when_possible
-    # - use_setters_to_change_properties # not yet tested
-    # - use_string_buffers # has false positives: https://github.com/dart-lang/sdk/issues/34182
-    # - use_to_and_as_if_applicable # has false positives, so we prefer to catch this by code-review
-    - valid_regexps
-    # - void_checks # not yet tested
+    # a few rules listed below are the ones we would like to exclude from flutter_lint package, for app_dart
+    # reasons for exclusions are provided in the comments to the right
+    avoid_print: false # we have necessary print calls in the code
```

Comment:
Can you link where? By now, they should be changed to use log

Reply:
![Screen Shot 2022-02-28 at 1 12 23 PM](https://user-images.githubusercontent.com/38773539/156059428-a49fbb1d-cdf9-471d-9ab7-42fdb82d4174.png)

Reply:
umm yeah I am not sure if logs have been created for each of these files. Some sample print statement looks like this
![Screen Shot 2022-02-28 at 1 13 50 PM](https://user-images.githubusercontent.com/38773539/156059883-fae94a92-230f-48e1-a4e2-77e98169590a.png)

Reply:
I think all of these are fine to switch with `log.fine`, you'll just need to add an import

Reply:
cool for sure

Reply:
Oh I think we actually need to keep the print statements, as they are eventually being directed by useLoggingPackageAdaptor() function in server.dart, to actual logging implementations such as setupAppEngineLogging()  function in logging_package_adaptor.dart. The print statements align with the examples listed in https://pub.dev/packages/logging, where print statements are eventually captured by log listeners

Reply:
package:logging should default to stdout if there's no other instances

Reply:
umm My understanding is that the logging is initialized once for all in server.dart

Reply:
Ah, I see what you mean. We can follow up with this in a different PR